recipes/tigervnc/meta.yaml

```
@@ -0,0 +1,92 @@
+{% set name = "tigervnc" %}
+{% set version = "1.12.0" %}
```

Try building [tigervnc=1.13.1](https://github.com/TigerVNC/tigervnc/releases/tag/v1.13.1)?

```suggestion
{% set version = "1.13.1" %}
```